src/app_functions_sdk_py/constants.py

```diff
@@ -40,3 +40,17 @@
 SECRET_CA_CERT = "cacert"
 
 DB_SQLITE = "sqlite"
+
+ENV_KEY_SECURITY_SECRET_STORE = "EDGEX_SECURITY_SECRET_STORE"
+ENV_KEY_DISABLE_JWT_VALIDATION = "EDGEX_DISABLE_JWT_VALIDATION"
+ENV_KEY_CONFIG_PROVIDER_URL = "EDGEX_CONFIG_PROVIDER"
+ENV_KEY_COMMON_CONFIG = "EDGEX_COMMON_CONFIG"
+ENV_KEY_USE_REGISTRY = "EDGEX_USE_REGISTRY"
+ENV_KEY_STARTUP_DURATION = "EDGEX_STARTUP_DURATION"
+ENV_KEY_STARTUP_INTERVAL = "EDGEX_STARTUP_INTERVAL"
+ENV_KEY_CONFIG_DIR = "EDGEX_CONFIG_DIR"
+ENV_KEY_PROFILE = "EDGEX_PROFILE"
+ENV_KEY_CONFIG_FILE = "EDGEX_CONFIG_FILE"
+ENV_KEY_FILE_URI_TIMEOUT = "EDGEX_FILE_URI_TIMEOUT"
+ENV_KEY_REMOTE_SERVICE_HOSTS = "EDGEX_REMOTE_SERVICE_HOSTS"
+ENV_EDGEX_MSG_BASE64_PAYLOAD = "EDGEX_MSG_BASE64_PAYLOAD"
```

Comment:
```suggestion
ENV_KEY_MSG_BASE64_PAYLOAD = "EDGEX_MSG_BASE64_PAYLOAD"
```
For naming consistency

Reply:
modified